sample/src/main/java/com/canhub/cropper/sample/MainActivity.kt

```diff
@@ -18,7 +18,6 @@ internal class MainActivity : AppCompatActivity() {
         binding.sampleCropImageView.setOnClickListener { SampleUsingImageView.newInstance().show() }
         binding.sampleCustomActivity.setOnClickListener { SampleCustomActivity.start(this) }
         binding.sampleCropImage.setOnClickListener { SampleCrop.newInstance().show() }
-        binding.sampleCropImageJava.setOnClickListener { SampleCropJava.newInstance().show() }
```

Comment:
Why remove this?

Reply:
I don't see much benefit of providing it. We should not be the one teaching people how to use Kotlin from Java.

Reply:
> We should not be the one teaching people how to use Kotlin from Java\

Why should we not show how to use the library in Java?

There are many requests to use the library in Java code, we have documentation only for this and is by far the most amount of issues I get in the repo.

I need a way to manually test if is working, in the case someone spot a bug in the library.

Reply:
> this and is by far the most amount of issues I get in the repo.

Seems like we have different motivations. I'll eventually get around of it and update it. In the meantime, I'll build a version myself.

> I need a way to manually test if is working, in the case someone spot a bug in the library.

And this can't be done using Kotlin only? What's a bug that is only targeting Java?

I'm just in favor of reducing maintenance cost here. We don't need similar things twice. Kotlin **and** Java.

Reply:
That said, if the community that uses this library wanna go in this direction, I will not oppose it. But we need someone to take ownership of the Java questions.